src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -82,22 +82,30 @@ def iot_hub_sku_list(client, hub_name, resource_group_name=None):
     return client.get_valid_skus(resource_group_name, hub_name)
 
 
-def iot_hub_consumer_group_create(client, hub_name, consumer_group_name, resource_group_name=None, event_hub_name='events'):
+def iot_hub_consumer_group_create(client, hub_name, consumer_group_name, resource_group_name=None, event_hub_name=None):
+    if event_hub_name is None:
```

Comment:
Suggest making this `event_hub_name='events'` then you don't need to keep having the if conditions.
Then, you can remove the text 'Default is events.' from `_params.py` and the default value will be added to the help automatically.
```
+register_cli_argument('iot hub consumer-group', 'event_hub_name', id_part='child_name',
+                      help='Event hub endpoint name. Default is events.')
```